packages/mdc-button/_mixins.scss

```diff
@@ -130,6 +130,15 @@
   }
 }
 
+@mixin mdc-button__icon_ {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  margin-right: 8px;
+  font-size: 16px;
+  vertical-align: middle;
```

Comment:
These icons don't actually look like they're vertically centered; their tops align, but they look like they're extending below the baseline, which is going to look awkward at least with all-caps text like our buttons typically have. I'm not sure what the design goal is here though - technically right now I think our button text vertical placement is centered disregarding anything hanging below the baseline.

Also, I'm wondering how this will interact with dense button, or is it not intended to ever be combined with that?

Reply:
Let me see if I could address this. The problem here is the button text is 14px (and 13px for dense button). However, the targeted icon side is 16px. I was not sure about the intention and may check with design tomorrow in the sign off.